docs/SOP/NewOntologyRequest.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This document should probably be simply `docs/SOP.md` and the New Ontology Request handling should be a section in it?

Reply:
ah, sure, i can do that. I'll fix it now

Reply:
done